src/core/react/ColorPicker.tsx

```diff
@@ -102,7 +119,22 @@ export const ColorPicker = React.memo((props: ColorPickerProps) => {
                         </div>,
                     )}
                 </div>
+
+                {colorsCurrentlyUsedInEditor(colorsCurrentlyUsed).length > 0 && <div>{texts.common.documentColors}</div>}
```

Comment:
It is personal preference, but I do not like these single line statements, because it is not easy to scan what is actually rendered. Just but the div to the next line.

Reply:
Fixed